packages/runtime/src/lib/load-program.ts

```diff
@@ -28,3 +28,31 @@ export function getProgram(fileName: string, scInitParam?: SCParams, logs = true
 		return pyOp.replaceTempValues(program, replaceParams);
 	}
 }
+
+
+/**
+ * returns program TEAL code from assetPath directory.
+ * @param fileName filename in assetPath. Must end with .teal OR .py
+ * @param assetPath path to directory that include filename
+ * @param scInitParam smart contract template parameters, used to set hardcoded values
+ * in .py or .teal smart contract.
+ * @param logs only show logs on console when set as true. By default this value is true
+ */
+ export function getProgramFromPath(fileName: string, assetPath: string,
+	scInitParam?: SCParams, logs = true): string {
+	const filePath = getPathFromDirRecursive(assetPath, fileName) as string;
+	const program = fs.readFileSync(filePath, "utf8");
```

Comment:
You should put assert/throw or any check if filePath exists or not before reading file.